src/core/modules/Asterism.cpp

```diff
@@ -88,7 +88,7 @@ bool Asterism::read(const QString& record, StarMgr *starMgr)
 				}
 				else
 				{
-					asterism[i]=starMgr->searchGaia(HP);
+					asterism[i]=starMgr->searchGaia(static_cast<int64_t>(HP));
```

Comment:
No need for a cast here. And `searchGaia` should accept a `StarId`.

Reply:
fixed